framework/doc/content/source/functions/PiecewiseLinearFromVectorPostprocessor.md

```diff
@@ -0,0 +1,21 @@
+# PiecewiseLinearFromVectorPostprocessor
```

Comment:
total rider commit. please let me know if it s not welcome

Reply:
I'd appreciate separation

Reply:
sounds good, I ll move the commit to a general doco PR